src/common/seed/images.ts

```diff
@@ -105,14 +118,19 @@ const generateDummyImage = () => ({
 	started_at: new Date(),
 	targetdevice: generateTargetDevice(),
 	thumb: generateRandomImage(),
-	build: generateBuild(),
+	build: generateBuild(statusIndex),
 });
 
 export const generateDummyImages = (length: number) => {
-	const images = Array.from(
+	const staticImages = Array.from(
 		{length},
+		(v, index) => generateDummyImage(index),
+	);
+
+	const restImages = Array.from(
+		{length: length - 3},
 		() => generateDummyImage(),
 	);
 
-	return images;
+	return (length < 4) ? staticImages : staticImages.concat(restImages);
```

Comment:
use a spread operator

Reply:
even better, move all this logic into a generateBuild function. generateDummyImages should simply return list of generateDummyImage